libs/gamify/data/src/lib/exports/types.ts

```diff
@@ -1,6 +1,7 @@
-import { Application, Badge, ApplicationUser, UserBadge } from '../../../generated';
+import { Application, Badge, ApplicationUser, UserBadge, FavoriteBadge } from '../../../generated';
```

Comment:
I often see "FavoriteBadge" or "IFavoriteBadge", what is the meaning of that? just curious

Reply:
The I prefix is used for clarifying that the type is an interface and not a class.